git_machete/cmd.py

```diff
@@ -718,15 +684,15 @@ def print_new_line(new_status: bool) -> None:
                     rmt: Optional[str] = self.git.inferred_remote_for_fetching_of_branch(b)
                     print_new_line(False)
                     if rmt:
-                        handle_untracked_branch(self.git, rmt, b)
+                        self.handle_untracked_branch(rmt, b)
```

Comment:
Shouldn't this method be private as well? 🤔

Reply:
well, i found this method useful when it's used from API. Someone may want to sync certain branch only and not traverse whole repository or its parts.